lms/static/my.css

```diff
@@ -1,3 +1,18 @@
+@media screen and (max-width: 30em) {
+  .exercise-id {
+    display: none !important;
```

Comment:
Try not to use `!important`. If needed, move `@media` queries to the end of the CSS document.